.babelrc

```diff
@@ -14,6 +14,7 @@
     "transform-class-properties",
     "transform-object-rest-spread",
     "lodash",
+    "syntax-dynamic-import",
```

Comment:
Something to note, this uses dynamic imports that @benvinegar introduced (but not merged) here: 
https://github.com/getsentry/sentry/compare/codesplit-notes#diff-e56633f72ecc521128b3db6586074d2cR16